.gitignore

```diff
@@ -72,6 +72,7 @@ tools/FAKE.Core
 tools/SourceLink.Fake
 tools/xunit.runners
 *.ncrunch*
+*.GhostDoc.xml
```

Comment:
Hmmm, I might want to start using GhostDoc again. It helps with the boilerplate ctor descriptions etc. What's in the .xml doc? Would we want to include this in the repo if we start using GhostDoc?